src/Corporation/ui/IndustryWarehouse.tsx

```diff
@@ -83,8 +83,10 @@ function WarehouseRoot(props: IProps): React.ReactElement {
   const mats = [];
   for (const matName of Object.keys(props.warehouse.materials)) {
     if (!(props.warehouse.materials[matName] instanceof Material)) continue;
-    // Only create UI for materials that are relevant for the industry
-    if (!isRelevantMaterial(matName, division)) continue;
+    // Only create UI for materials that are relevant for the industry or in stock
+    const isInStock = props.warehouse.materials[matName].qty > 0;
+    const isRelevant = isRelevantMaterial(matName, division);
+    if (!isInStock && !isRelevant) continue;
```

Comment:
wouldn't this stop people from wanting to make an export, say they have 0 but want it to export next tick?

Reply:
This is applied to unrelevant materials only. People won't have this kind of materials in their warehouses anyway unless they accidentally exported them from another division by mistake.

Reply:
ah good point 👍